packages/react-ui-core/src/Gmap/FreeDrawLayer.js

```diff
@@ -125,13 +131,20 @@ export default class FreeDrawLayer extends PureComponent {
 
     this.events = {
       ...setupEvents(map, GMAP_EVENTS, {
-        onMouseMove: e => this.polyline.getPath().push(e.latLng),
-        onTouchStart: e => this.polyline.getPath().push(e.latLng),
+        onMouseDown: () => { this.mouseDown = true },
+        onTouchStart: () => { this.mouseDown = true },
+        onMouseMove: e => this.handleMouseMove(e),
+        onTouchMove: e => this.handleMouseMove(e),
```

Comment:
This should be:

onTouchMove: this.handleMouseMove,

Reply:
👍